src/journeys_utils.js

```diff
@@ -857,14 +857,18 @@ journeys_utils._handleJourneyDismiss = function(eventName, storage, banner, temp
 }
 
 journeys_utils._getPageviewMetadata = function(options, additionalMetadata) {
-	return utils.merge({
+	var pageviewMetadata = utils.merge({
 		"url": options && options.url || utils.getWindowLocation(),
 		"user_agent": navigator.userAgent,
 		"language": navigator.language,
 		"screen_width": screen.width || -1,
 		"screen_height": screen.height || -1,
-		"window_device_pixel_ratio": window.devicePixelRatio || 1
+		"window_device_pixel_ratio": window.devicePixelRatio || 1,
```

Comment:
Is this default safe/why not -1 if not found?

Reply:
for older browsers that don't support devicePixelRatio, we fallback to 1 which is classic or standard display with zoom at 100%. You rarely see any devices with device pixel ratio in negative, https://www.mydevice.io/#tab1
This is pretty old code and not touched as part of this PR  and we can evaluate based on our browser support, if we need these fallbacks to continue.